repository-data/webfiles/src/main/resources/site/freemarker/nationalindicatorlibrary/indicator.ftl

```diff
@@ -89,13 +89,16 @@
         
 
         <section id="resources" class="push-double--bottom">
-            <h3>Resources</h3>
-            <ul>
-                <li><a href="#">Sample file</a></li>
-                <li><a href="#">Sample file</a></li>
-                <li><a href="#">Sample file</a></li>
-                <li><a href="#">Sample file</a></li>
-            </ul>
+            <h2><strong><@fmt.message key="headers.resources"/></strong></h2>
+            <#if indicator.attachments?has_content>
+                 <ul data-uipath="nil.indicator.resources">
+                    <#list indicator.attachments as attachment>
+                        <li class="attachment">
+                            <a title="${attachment.text}" href="<@hst.link hippobean=attachment.resource/>" onClick="logGoogleAnalyticsEvent('Download attachment','Publication','${attachment.resource.filename}');">${attachment.text}</a>
```

Comment:
Suggest changing to `... 'Download attachment','Indicator' ...` or similar to distinguish these from publications in Google Analytics.

Reply:
Good spot.